engine/src/main/java/org/destinationsol/world/generators/FeatureGenerator.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2020 The Terasology Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.destinationsol.world.generators;
+
+/**
+ * This class represents the framework
```

Comment:
Can you elaborate what framework? While it may be clear from context now, think about a person who is designing their own feature generator, without having followed the creation of this code. What will help that person understand how to use this code?